security/pkg/pki/util/keycertbundle.go

```diff
@@ -233,6 +233,24 @@ func (b *KeyCertBundleImpl) ExtractCACertExpiryTimestamp() (float64, error) {
 	return extractCertExpiryTimestamp("CA cert", b.GetCertChainPem())
 }
 
+// TimeBeforeNextRotate returns time duration to the next cert rotation
```

Comment:
The comment is basically just repeating the function name, update it to be more explicit that it returns the smaller duration between certBytes and workloadTTL, and also end with `.`.

And the function could also be renamed to be more generic as it returns the smaller duration between the two arguments.